src/views/Ifos/components/IfoVesting/index.tsx

```diff
@@ -26,16 +27,13 @@ const VestingCardBody = styled(CardBody)`
   overflow-y: auto;
   max-height: 570px;
   padding-bottom: 0;
+  border-radius: 24px;
 `
 
 const TokenInfoContainer = styled.div`
   > div {
     margin-bottom: 20px;
   }
-
-  > :last-child {
-    margin-bottom: 0px;
-  }
 `
```

Comment:
don't delete this
![Screenshot 2022-08-05 at 2 43 59 PM](https://user-images.githubusercontent.com/98292246/183017984-2b25d6be-1f95-4b95-976d-d9b958987a23.png)